content/_guides/catalyst.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Catalysing and De-escalation
+category: chanop
+---
+
+People often use Libera.Chat to get help with something on their computer.
+Technical support channels are one of the things we're here for!
+
+If you have ever had a technical problem, you know how overwhelming it can be.
+Sometimes people seeking help will get very frustrated and might have a temper
+or be rude. Patient and efficient help can be vital in avoiding conflict in
+these situations.
+
+This guide will outline some good practices for both operators and helpers for
+interacting with people. It will also discuss some common situations and ideal
+ways of responding.
+
+## Only be "opped up" when necessary
+
+Most channels have a select few operators who are responsible for managing the
+culture of the channel. These are the people with `+o` flags. Most clients
```

Comment:
I wonder if we should make a clearer distinction between ChanServ flags and IRC modes, since there are often confusion about it. Maybe as simple as spelling out "people with the ChanServ flag +o" and later being voice (having mode +v) or similar